http_general/test_headers.py

```diff
@@ -332,6 +334,11 @@ class TestHeadersParsing(tester.TempestaTest):
         },
     ]
 
+    def __is_hbp(self, tr):
+        if tr.lower() == "connection" or tr.lower() == "keep-alive" or tr.lower() == "proxy-connection" or tr == "HbpHeader1" or tr == "HbpHeader2":
+            return True
+        return False
```

Comment:
What does `hbp` mean? Please do not use short abbreviations. It's not that obvious.
```suggestion
        if tr.lower() in ["connection" , "keep-alive", "proxy-connection", "HbpHeader1", "HbpHeader2"]:
            return True
        return False
```